tests/ftests/052-sudo-set_uid_gid_v1.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: LGPL-2.1-only
+#
+# Test to set the uid/gid on a cgroup using the python bindings
+#
+# Copyright (c) 2022 Oracle and/or its affiliates.
+# Author: Tom Hromatka <tom.hromatka@oracle.com>
+#
+
+from cgroup import Cgroup as CgroupCli, CgroupVersion
+from libcgroup import Cgroup, Version
+import consts
+import ftests
+import utils
+import sys
+import os
+
+CGNAME = '052setuidgid'
+CONTROLLER = 'cpu'
+TASKS_UID = 1234
+TASKS_GID = 5678
+CTRL_UID = 3456
+CTRL_GID = 7890
+
+
+def prereqs(config):
+    result = consts.TEST_PASSED
+    cause = None
+
+    if CgroupVersion.get_version(CONTROLLER) != CgroupVersion.CGROUP_V1:
+        result = consts.TEST_SKIPPED
+        cause = 'This test requires cgroup v1'
+
+    return result, cause
+
+
+def setup(config):
+    return consts.TEST_PASSED, None
+
+
+def test(config):
+    result = consts.TEST_PASSED
+    cause = None
+
+    cg = Cgroup(CGNAME, Version.CGROUP_V1)
+    cg.set_uid_gid(TASKS_UID, TASKS_GID, CTRL_UID, CTRL_GID)
+    cg.add_controller(CONTROLLER)
+    cg.create(ignore_ownership=False)
+
+    tasks_path = os.path.join(CgroupCli.get_controller_mount_point(CONTROLLER), CGNAME,
+                              'tasks')
+
+    tasks_uid = utils.get_file_owner_uid(config, tasks_path)
+    if tasks_uid != TASKS_UID:
+        result = consts.TEST_FAILED
+        cause = 'Expected tasks owner to be {} but it\'s {}'.format(TASKS_UID, tasks_uid)
+
+    tasks_gid = utils.get_file_owner_gid(config, tasks_path)
+    if tasks_gid != TASKS_GID:
+        result = consts.TEST_FAILED
+        cause = 'Expected tasks group to be {} but it\'s {}'.format(TASKS_GID, tasks_gid)
+
+    ctrl_path = os.path.join(CgroupCli.get_controller_mount_point(CONTROLLER), CGNAME,
+                             'cgroup.procs')
+
+    ctrl_uid = utils.get_file_owner_uid(config, ctrl_path)
+    if ctrl_uid != CTRL_UID:
+        result = consts.TEST_FAILED
+        cause = 'Expected cgroup.procs owner to be {} but it\'s {}'.format(CTRL_UID, ctrl_uid)
+
+    ctrl_gid = utils.get_file_owner_gid(config, ctrl_path)
+    if ctrl_gid != CTRL_GID:
+        result = consts.TEST_FAILED
+        cause = 'Expected cgroup.procs group to be {} but it\'s {}'.format(CTRL_GID, ctrl_gid)
+
+    return result, cause
+
+
+def teardown(config):
+    CgroupCli.delete(config, CONTROLLER, CGNAME)
+
+    return consts.TEST_PASSED, None
```

Comment:
this return is not used in `main` and the return is True always, should this be removed?